docs/user-guide/wiki/README.md

```diff
@@ -57,6 +57,12 @@ As CKEditor5 currently does not provide support for code blocks, OpenProject can
 
 The GFM extension of the CommonMark specs adds a definition for table syntax which the CKEditor build of OpenProject supports. This definition requires all tables to have a heading row. For tables created with CKEditor without heading rows, a HTML table is output instead. This matches the behavior of, e.g., GitHub.
 
+### Emojis
```

Comment:
Should this section be added also to the Work packages documentation for comments and descriptions?

Reply:
@marcalcobe yes, good point. I've added the info both to wp descriptions and comments sections